ting-bill-split.go

```diff
@@ -362,35 +364,166 @@ func createBillsFile(path string) {
 	}
 }
 
-func main() {
-	fmt.Printf("Ting Bill Splitter\n\n")
+func isFileMatch(fileName string, nameTerm string, ext string) bool {
+	r := regexp.MustCompile(`(?i)^[\w-]*` + nameTerm + `[\w-]*$`)
+
+	if ext != "" {
+		r = regexp.MustCompile(`(?i)^[\w-]*` + nameTerm + `[\w-]*(\.` + ext + `)$`)
+	}
+
+	return r.MatchString(fileName)
+}
+
+func parseDir(path string) {
+	files, err := ioutil.ReadDir(path)
+
+	var billFile *os.File
+	var minFile *os.File
+	var msgFile *os.File
+	var megFile *os.File
 
-	billPtr := flag.String("bills", "", "filename for bills toml - ex: -bills=\"bills.toml\"")
-	minPtr := flag.String("minutes", "", "filename for minutes csv - ex: -minutes=\"minutes.csv\"")
-	msgPtr := flag.String("messages", "", "filename for messages csv - ex: -messages=\"messages.csv\"")
-	megPtr := flag.String("megabytes", "", "filename for megabytes csv - ex: -megabytes=\"megabytes.csv\"")
+	if err != nil {
+		log.Fatal(err)
+	}
 
-	badParam := false
-	paramMap := map[string]*string{
-		"bills":     billPtr,
-		"minutes":   minPtr,
-		"messages":  msgPtr,
-		"megabytes": megPtr,
+	for _, file := range files {
+		if !file.IsDir() {
```

Comment:
Change this to 
```
if file.IsDir() {
    continue
}
```

and dedent the rest of this. de-indent? Un-indent? Undent? Back it up! lol

Reply:
BACKED UP
*airhorn  and beeping sounds*